alacritty/src/renderer/platform.rs

```diff
@@ -81,21 +81,31 @@ pub fn create_gl_context(
     gl_config: &Config,
     raw_window_handle: Option<RawWindowHandle>,
 ) -> GlutinResult<NotCurrentContext> {
-    let context_attributes = ContextAttributesBuilder::new()
-        .with_context_api(ContextApi::OpenGl(Some(Version::new(3, 3))))
-        .build(raw_window_handle);
-
-    unsafe {
-        if let Ok(gl_context) = gl_display.create_context(gl_config, &context_attributes) {
-            Ok(gl_context)
-        } else {
-            let context_attributes = ContextAttributesBuilder::new()
-                .with_profile(GlProfile::Compatibility)
-                .with_context_api(ContextApi::OpenGl(Some(Version::new(2, 1))))
-                .build(raw_window_handle);
-            gl_display.create_context(gl_config, &context_attributes)
-        }
+    let mut profiles = [
+        ContextAttributesBuilder::new()
+            .with_context_api(ContextApi::OpenGl(Some(Version::new(3, 3))))
+            .build(raw_window_handle),
+        // Try gles before OpenGL 2.1 as it tends to be more stable.
```

Comment:
Curious why you say this? Is this also the case on macOS for example?

Reply:
macOS is always a OpenGL 3.3 renderer.

Reply:
We might swap them, it just far less easier to achieve CTS conformance on gles then on OpenGL 2.1, and that's what mostly tested.